AlexNet.js

```diff
@@ -195,12 +195,12 @@ function AlexNet() {
             if (showConvDims && index < architecture.length - 1) {
 
                 // Conv Dims
-                sprite = makeTextSprite(layer['filterWidth'].toString());
-                sprite.position.copy( conv_object.position ).sub( new THREE.Vector3( convFn(layer['filterWidth'])/2, -3, depthFn(layer['depth'])/2 ) );
+                sprite = makeTextSprite(layer['filterHeight'].toString());
+                sprite.position.copy( conv_object.position ).sub( new THREE.Vector3( convFn(layer['filterHeight'])/2, -3, depthFn(layer['depth'])/2 ) );
```

Comment:
I think you need to change like this. Because `height` text should be placed to the right, it needs to use `width` size. 
```suggestion
                sprite.position.copy( conv_object.position ).sub( new THREE.Vector3( convFn(layer['filterWidth'])/2, -3, depthFn(layer['depth'])/2 ) );
```